app/models/accountModel.js

```diff
@@ -0,0 +1,122 @@
+import pool from '../db';
+
+export default class AccountModel {
+  static async create(req, res, userId) {
+    const owner = userId;
+    const status = 'active';
+    const allowedTypes = ['savings', 'current'];
+    const accountNo = Math.floor(Math.random() * 1000000000);
+    const createdOn = new Date();
+    const { type, openingBalance } = req.body;
+    const createQuery = `INSERT INTO accounts(accountNo, createdOn, owner, type, status, openingBalance)
+                            VALUES ($1, $2, $3, $4, $5, $6) 
+                            RETURNING accountNo, createdOn, owner, type, openingBalance, status`;
+    const values = [accountNo, createdOn, owner, type.trim(), status, openingBalance];
+    if (!allowedTypes.includes(type)) {
+      return res.status(400).send({
+        status: 400,
+        error: 'invalid account type, must be either "savings" or "current"',
+      });
+    }
+    try {
+      const { rows } = await pool.query(createQuery, values);
+      return rows;
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  static async getAccount(req, res) {
+    const query = 'SELECT * FROM accounts WHERE accountNo = $1';
+    const values = [req.params.accountNumber];
+    try {
+      const { rows } = await pool.query(query, values);
+      if (!rows[0]) {
+        return res.status(200).send({
```

Comment:
200 on this line and 204 on the next. fix this

Reply:
okay will fix it